src/flows/recommendation_api/corpus_candidate_sets/collections_by_recency.py

```diff
@@ -15,15 +15,30 @@
 RECENT_COLLECTIONS_CANDIDATE_SET_ID = "92af3dae-25c9-46c3-bf05-18082aacc7e1"
 
 EXPORT_COLLECTIONS_CANDIDATE_SET_SQL = """
-SELECT 
+WITH recent_collections AS (
+  SELECT
+    IFNULL(s.SCHEDULED_CORPUS_ITEM_SCHEDULED_AT, a.REVIEWED_CORPUS_ITEM_CREATED_AT) as recency,
+    s.SCHEDULED_SURFACE_IANA_TIMEZONE,
+    a.*
+  FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" a
+  LEFT JOIN "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS" s ON (
+    s.approved_corpus_item_external_id = a.approved_corpus_item_external_id
+    AND s.SCHEDULED_CORPUS_ITEM_SCHEDULED_AT < current_timestamp()
+    AND s.SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
+  )
+  WHERE a.IS_COLLECTION
+  AND a.CORPUS_REVIEW_STATUS = 'recommendation'
+  AND a.LANGUAGE = 'EN'
+  AND recency > DATEADD("day", %(MAX_AGE_DAYS)s, current_timestamp())
```

Comment:
this should probably be something like `BETWEEN DATEADD("day", %(MAX_AGE_DAYS)s, current_timestamp()) AND current_timestamp()` to avoid showing items that are scheduled in the future as is a common practice

Reply:
Those items are removed by the condition on line 26.